src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,10 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+	
+	
+	
+	String getPrintableString();
```

Comment:
Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.